sw/device/boot_rom/rom_crt.S

```diff
@@ -75,18 +81,39 @@ _reset_start:
 _start:
   .globl _start
 
+  // Enable entropy complex - this is not the full enable
+  li   a0, TOP_EARLGREY_ENTROPY_SRC_BASE_ADDR
+  li   t0, 0x2
+  sw   t0, ENTROPY_SRC_CONF_REG_OFFSET(a0)
```

Comment:
could you break each store into a `li/li/sw` stanza? Just so it's clear that it's a "store immediate to immediate".

Reply:
is this what you meant..?

Reply:
Yup